app/providers/search/page.tsx

```
@@ -0,0 +1,122 @@
+    if (query) params.append('q', query);
+    if (category) params.append('category', category);
+    
+    const res = await fetch(`${baseUrl}/providers?${params}`, { next: { revalidate: 60 } });
+    
+    if (!res.ok) {
+      throw new Error('Failed to search providers');
+    }
+    
+    const data = await res.json();
+    // Map snake_case API fields to camelCase for the client component
+    return (data.providers || []).map((provider: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)